src/utils/keyStore.ts

```diff
@@ -0,0 +1,45 @@
+const LS_KEY_STORE = 'dotflow-key-store';
+
+interface IKeyStore {
+  [key: number]: string;
+}
+
+class KeyStore {
```

Comment:
We should separate keys per users, since the way it is now if someone uses the same browser with two accounts some overlapping can happen. That should probably be a separate PR though.